testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java

```diff
@@ -56,6 +56,8 @@ public static void configureTempto(Environment.Builder builder, ResourceProvider
     {
         builder.configureContainer(TESTS, dockerContainer -> {
             Path path = configDir.getPath("tempto-configuration.yaml");
+            // The suffix should identify the EnvironmentExtender that contributed the config file so we use the parent
+            // directory name instead of product test environment name (which can be same for multiple EnvironmentExtenders)
```

Comment:
This is a bit long comment for me.

> The suffix should identify the EnvironmentExtender

That is the best scenario, however it requires to follow the convention that parent directory has same name as extender. Which might be not always true.

I would just say that `the suffix uses parent directory to help identity the actual tempto configuration file`.

Reply:
Done.